Capstone_Project/Inferential_Statistics.ipynb

```diff
@@ -0,0 +1,375 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x206ebc300f0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.distplot(np.log(df_other)) #taking log scale to get clear view of ditribution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(2.7944851023093413, 0.407980146879094)"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "h0_mean = 0 #from null hypothesis the mean would be 0\n",
+    "mean_diff = izakaya_mean_visitors - other_mean_visitors\n",
+    "sigma_diff = izakaya_std - other_std\n",
+    "mean_diff, sigma_diff"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Using student's t-statistic(Pooled variances) as the variance of population is unknown\n",
+    "#### First find the common standard deviation ($s_{p}$)that can be estimated by the pooled standard deviation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Degrees of Freedom :  128582\n",
+      "Common Standard Deviation :  23.4033971111\n",
+      "t-statistic :  25.7849851865\n"
+     ]
+    }
+   ],
+   "source": [
+    "df = len(df_other) - len(df_izakaya)  + 2 #degree of freedom\n",
+    "sp = np.sqrt( ( ( len(df_izakaya) - 1 ) * (izakaya_std ** 2 ) + ( len(df_other) - 1 ) * (other_std ** 2 ) ) / df ) # common standard deviation\n",
+    "t = (mean_diff - h0_mean) / (sp * ( np.sqrt((1/len(df_izakaya))+(1/len(df_other)))) ) #t-statistic\n",
+    "print(\"Degrees of Freedom : \",df)\n",
+    "print(\"Common Standard Deviation : \",sp)\n",
+    "print(\"t-statistic : \",t)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "P-Value :  0.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "p_Val = 1 - stats.t.cdf(t,df=df)\n",
+    "print(\"P-Value : \",p_Val)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<b> Here the p-Value is less than than the significant level $\\alpha$, so Rejecting the null hypothesis $H_{0}$ the mean vistor count for izakaya and other genres are same </b>"
```

Comment:
@nishalpattan : good idea in making this hypothesis test. Could I suggest to rephrase the result of the test in more "plain english"? For instance: "there is (or there is not, according to the result) statistical evidence that the mean of ..... "
What do you think?